src/Akeneo/Tool/Bundle/MeasureBundle/PublicApi/FindMeasurementFamilies.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Tool\Bundle\MeasureBundle\PublicApi;
```

Comment:
```suggestion
namespace Akeneo\Tool\Bundle\MeasureBundle\ServiceApi;
```
Let's benefit from this move to use the new standard for ServiceApi naming 😉